### PR TITLE
Tests For Big number and SHA3

### PR DESCRIPTION
**Pull request by rihp**
This tests several calls for the bignumber wrapper.

An finds a new error when testing the SHA3 wrapper, related to `wasmtime` and interfaces:

> Unsupported feature: Support for interface types has temporarily been removed from wasmtime.   For more information about this temoprary you can read on the issue online:   https://github.com/bytecodealliance/wasmtime/issues/1271    and for re-adding support for interface types you can see this issue:   https://github.com/bytecodealliance/wasmtime/issues/677

Full error trace:
image

# This comment (https://github.com/bytecodealliance/wasmtime/issues/677#issuecomment-1024087373) sheds a light on a potentially solution for the issue

### CODE REVIEW

**Comment by rihp** [comment]
I've added a placeholder for all the tests for the Sha3 wrapper, regardless of them failing at the moment, for the same reason detailed above!

File: packages/polywrap-client/tests/test_bignumber.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    }
+    options = InvokerOptions(uri=uri, method="method", args=args, encode_result=False)
+    result = await client.invoke(options)
+    print(result)
+    assert result.result == str(123.123*55.5*10.001*4)
+
+
+# async def test_invoke_bignumber_with_2args_and_2props_complex_numbers():
+#     client = PolywrapClient()
+#     # BigNumber wrapper schema - https://wrappers.io/v/ipfs/Qme2YXThmsqtfpiUPHJUEzZSBiqX3woQxxdXbDJZvXrvAD
```
Comment (Niraj-Kamdar) [inline packages/polywrap-client/tests/test_bignumber.py:91]: No polywrap doesn't support complex numbers yet and isn't planning to do so anytime soon.

Comment (Niraj-Kamdar) [reply packages/polywrap-client/tests/test_bignumber.py:91]: I don't see a real need for this yet but correct me if I am wrong

File: packages/polywrap-client/tests/test_sha3.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+client = PolywrapClient()
+uri = Uri(f'fs/{Path(__file__).parent.joinpath("cases", "simple-invoke", "wrapperSHA3.wasm").absolute()}')
+args = {"message": "hello polywrap!"}
+
+async def test_invoke_sha3_512():
+    options = InvokerOptions(uri=uri, method="sha3_512", args=args, encode_result=False)
+    result = await client.invoke(options)
+    print(result)
+    assert result.result == False
```
Comment (Niraj-Kamdar) [inline packages/polywrap-client/tests/test_sha3.py:16]: Why result is False in every case?

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]
This shouldn't have readme changes here and only contain relevant changes

Regarding

File: packages/polywrap-client/tests/test_sha3.py (hunk 2 above)
Comment (rihp) [reply packages/polywrap-client/tests/test_sha3.py]: Updated the tests so it includes the correct assertions for almost every unit test. They still cant pass as green due to the `wasmtime` error mentioned above in the PR. 

also missing the right assertion for `test_invoke_buffer_keccak_256`

**Comment by rihp** [comment]
- [ ] Merge newest changes
- [ ] Skip the Sha3 errors
- [ ] change from draft to PR

**Comment by rihp** [comment]
Closing this PR in favor of #33

**Pull request closed without merging**